brax/physics/forces.py

```diff
@@ -74,12 +83,22 @@ def get(config: config_pb2.Config,
   dofs = {j.name: len(j.angle_limit) for j in config.joints}
   current_index = sum([dofs[a.joint] for a in config.actuators])
 
-  act_indices = []
-  for _ in config.forces:
-    act_index = tuple(range(current_index, current_index + 3))
-    act_indices.append(act_index)
-    current_index += 3
-  if act_indices:
-    return [Thruster(config.forces, body, act_indices)]
+  force_indices = []
+  torque_indices = []
+  for force in config.forces:
+    pos_mask = 1. * jp.logical_not(vec_to_arr(force.frozen.position))
+    force_dof = int(sum(pos_mask))
+    force_act_index = tuple(range(current_index, current_index + force_dof))
+    force_indices.append(force_act_index)
+    current_index += force_dof
+
+    rot_mask = 1. * jp.logical_not(vec_to_arr(force.frozen.rotation))
+    torque_dof = int(sum(rot_mask))
+    torque_act_index = tuple(range(current_index, current_index + torque_dof))
+    torque_indices.append(torque_act_index)
+    current_index += torque_dof
```

Comment:
Make sure you add `frozen: all` fields to the rotation fields on the forces in the inverted_pendulum and inverted_double_pendulum environments

Reply:
+1

Reply:
Added ```frozen { position { x: 0.0 y: 1.0 z: 1.0} rotation { x: 1.0 y: 1.0 z: 1.0 } ``` to both so only the needed axis works. I confirmed both environments produce stable behave and training curves look similar.

Side question. There is no convention to say ```frozen { rotation : all }``` that I can see